string label
int32 channel # byte index
